runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/CoordinateClue.java

```diff
@@ -168,6 +168,7 @@
 		.put(new WorldPoint(3051, 3736, 0), "East of the Wilderness Obelisk in 28 Wilderness.")
 		.put(new WorldPoint(2316, 3814, 0), "West of Neitiznot, near the bridge.")
 		.put(new WorldPoint(2872, 3937, 0), "Weiss.")
+		.put(new WorldPoint(2835, 2998, 0), "Northeast corner of the Island of Stone.")
```

Comment:
This should probably not be in this pr :)

Reply:
Uhh.
So basically I made a first change in a master branch.
Then made a new branch that already had the first change.
Okay, soo, I should probably just delete the line in this branch, then fix my master in future...

Thanks for the catch.